modules/sde-common/src/main/java/org/eclipse/tractusx/sde/common/entities/PolicyModel.java

```diff
@@ -0,0 +1,57 @@
+/********************************************************************************
+ * Copyright (c) 2022, 2023, 2024 T-Systems International GmbH
```

Comment:
This file is new, it contains old copy right year

Reply:
Updated all copyrights headers. Please review again